SampleWebApp/wwwroot/.placeholder

```diff
@@ -0,0 +1 @@
+��
```

Comment:
Do we need that file? What for?

Reply:
I'm not sure what put it there.  I was thinking maybe it was so git would create that folder.  I did a search for that file and it looks like maybe `OrchardCore.Application.Cms.Targets` is what creates it:
https://gitter.im/OrchardCMS/OrchardCore?at=5e276da76ced65043b5c25ba

Reply:
If it's generated, can we git ignore and remove it?

Reply:
I removed both of those folders and rebuilt the solution and it created them again so It seems they don't need to be in git.  I removed and ignored the .placeholder files.